### DISCUSSION TITLE
What's the best way to correlate helm version with werf version?

### DISCUSSION THREAD

**Discussion by andre-perrot** [werf/werf#5996] [General]
Trying to ensure that the version of werf I'm using is on helm v3.8.2. Any tips would greatly be appreciated! Thanks.

**Comment by alexey-igrychev** [comment]
Hi! 👋🏼

werf extends Helm in its fork, the current base version of Helm being [v3.14.0-rc.1](https://github.com/helm/helm/releases/tag/v3.14.0-rc.1). To determine the specific Helm version used, consult the [go.mod](https://github.com/werf/werf/blob/main/go.mod#L396) file.

You can also run the `werf helm version` command for some details:
```shell
$ werf helm version           
version.BuildInfo{Version:"v3.14", GitCommit:"", GitTreeState:"", GoVersion:"go1.21.6"}
```

**Reply by andre-perrot** [reply]
Thank you. Somehow I missed the binaries included in the actual release!
